package.json

```diff
@@ -43,7 +43,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
+    "@babel/core": "^7.6.2",
```

Comment:
you seemed to have downgraded some dependencies :s

Reply:
Fixed!